docs/system-admin-guide/files/external-file-storages/health-status/README.md

```diff
@@ -98,14 +96,14 @@ the [system admin guide](../../../../system-admin-guide/integrations/nextcloud/#
+| ERR_OIDC_TOKEN_ACQUISITION_FAILED | No token could be acquired for accessing the file storage. | The OpenID Connect (OIDC) setup doesn't have the necessary audience, nor does it provide the possibility to exchange tokens.                                                                                          | Please check if you configured the correct file storage audience in OpenProject.                                                                               |
+| ERR_OIDC_TOKEN_EXCHANGE_FAILED    | The token exchange failed.                                 | The OIDC provider might not allow token exchange, or the exchange failed on the given audience.                                                                                                                       | Please check if you configured the correct file storage audience in OpenProject, and if the OIDC identity provider allows exchanging tokens for that audience. |
+| ERR_OIDC_TOKEN_REFRESH_FAILED     | The expired access token could not get refreshed.          | The OIDC provider failed to refresh the token. This might happen due to an expired or revoked refresh token.                                                                                                          | Please check the configuration of the OIDC identity provider of refresh tokens for misconfiguration.                                                           |
+| WRN_OIDC_NON_OIDC_USER            | The user is not a OIDC user.                               | The current user is provisioned to OpenProject, but not by an OIDC identity provider.                                                                                                                                 | Log in to an admin user with the correct OIDC identity provider, the same one used by the file storage, and rerun the check.                                   |
+| WRN_OIDC_NON_PROVISIONED_USER     | The user is not a provisioned user.                        | The current user is not provisioned by an OIDC identity provider.                                                                                                                                                     | Log in to an admin user with the correct OIDC identity provider, the same one used by the file storage, and rerun the check.                                   |
+| WRN_OFFLINE_ACCESS_SCOPE_MISSING  | The OIDC provider has no `offline_access` scope.           | The OIDC provider was configured without `offline_access` scope. While this does not cause the integration to break, this scope ensures, that refresh tokens to not expire and OpenProject can maintain a connection. | Add the scope `offline_access` to the OIDC provider configuration.                                                                                             |
```

Comment:
Suggesting to clarify that the problem is how we configure the provider, but also that this is not a guaranteed way to fix it:

```suggestion
| WRN_OFFLINE_ACCESS_SCOPE_MISSING  | `offline_access` scope is not requested from the OIDC provider.           | The OIDC provider was configured without `offline_access` scope. While this does not cause the integration to break, this scope usually ensures, that refresh tokens do not expire and OpenProject can maintain a connection. | Make sure that refresh tokens issued by the identity provider do not expire. Adding the scope `offline_access` to the OIDC provider configuration is one common way to do it. |
```